src/mcpil.py

```diff
@@ -328,6 +357,8 @@ def play_tab(parent):
             title = ttk.Label(tab, text='Minceraft Pi Launcher')
         else:
             title = ttk.Label(tab, text='Minecraft Pi Launcher')
+
+    title.tooltip = ToolTip(title, text="This is the title, what are you gazing at?\nThere's a small chance it will show something else than this plain title...")
```

Comment:
Unnecessary imo

Reply:
Come on, don't you like jokes?

Reply:
It's the easter eggs that make things interesting :)

Reply:
Yeah but it’s not really an Easter egg

Reply:
Also, new lines in tooltips is ugly. They should be short, not a whole paragraph